src/main/java/org/openstreetmap/atlas/checks/event/EventService.java

```diff
@@ -96,4 +96,17 @@ public void complete()
     {
         this.eventBus.register(processor);
     }
+
+    /**
+     * Unregisters given {@link Processor}
+     *
+     * @param <T>
+     *            processor event type that is going to be registered
+     * @param processor
+     *            {@link Processor} to unregister
+     */
+    public <T extends Event> void unRegister(final Processor<T> processor)
```

Comment:
I'd go with deRegister, for grammatical correctness, but fine with leaving as is.